README.md

```diff
@@ -67,6 +67,22 @@ route_key = "hello"
 end)
 ```
 
+You can also use named connection like this:
+
+``` elixir
+options = %{
+  url: "amqp://rabbitmq:5672",
+  connection_id: :publisher,
+  exchange: "test-exchange",
+  routing_key: "test-messages",
+}
+
+Tackle.publish("Hi!", options)
```

Comment:
Can you add a test that verifies this? It should probably publish a few times in a row and check the number of open connections.